arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md

```diff
@@ -61,47 +64,46 @@ struct Config {
-| `MaxInitCodeSize`           | Similar to `MaxCodeSize`, defines the maximum size for your Orbit chain's **initialization** code. e.g. Ethereum mainnet limit is 49,152 Bytes. |
+| `MaxInitCodeSize`           | Similar to `MaxCodeSize`, defines the maximum size for your Orbit chain's **initialization** code. For example, the Ethereum mainnet limit is 49,152 Bytes. |
 
-For an AnyTrust chain, you need to set the `DataAvailabilityCommittee` to **true**. This setting is crucial as it indicates the chain's reliance on a committee for data availability, which is a hallmark of the AnyTrust model.
+For an AnyTrust chain, you need to set the `DataAvailabilityCommittee` to **true**. This setting is crucial as it indicates the chain's reliance on a committee for data availability.
 
-Here’s an example of how to configure the `chainConfig` for an AnyTrust chain using the Orbit SDK:
+Here's an example of how to configure the `chainConfig` for an AnyTrust chain using the Orbit SDK:
 
 ```js
-import { prepareChainConfig } from '@arbitrum/orbit-sdk';
+import { prepareChainConfig } from "@arbitrum/orbit-sdk";
 
 const chainConfig = prepareChainConfig({
   chainId: Some_Chain_ID,
-  arbitrum: { InitialChainOwner: deployer_address, DataAvailabilityCommittee: true },
+  arbitrum: {
+    InitialChainOwner: deployer_address,
+    DataAvailabilityCommittee: true,
+  },
 });
 ```
 
-In this example, you set up the chain configuration with a specific `chainId`, the `InitialChainOwner` as the deployer's address, and importantly, you configure the `DataAvailabilityCommittee` as `true`. This configuration ensures that your Orbit chain is set up as an AnyTrust chain, utilizing the unique features and operational model of the AnyTrust system within the Arbitrum Orbit framework.
-
-These parameters tailor the chain to your specific requirements and operational context.
-
-The Orbit SDK has a helper to execute this part:
+In this example, you need to set up the chain configuration with a unique `chainId`, the `InitialChainOwner` as the deployer's address, and, importantly, you must set the `DataAvailabilityCommittee` as `true`.
 
-For an easier config preparation, the Orbit SDK provides the `prepareChainConfig` API, which takes config parameters as arguments and returns a `chainConfig` `JSON` string. Any parameters not provided will default to standard values, which are detailed in the [Orbit SDK documentation](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/1f251f76a55bc1081f50938b0aa9f7965660ebf7/src/prepareChainConfig.ts#L3-L31).
+The Orbit SDK has a helper to execute this part: the `prepareChainConfig` API, which takes config parameters as arguments and returns a `chainConfig` `JSON` string. Any parameters not provided will default to standard values, which are detailed in the [Orbit SDK Repository](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/1f251f76a55bc1081f50938b0aa9f7965660ebf7/src/prepareChainConfig.ts#L3-L31).
```

Comment:
Any particular reason `Repository` is capitalized here?

Reply:
No reason, suggestion merged.